.husky/pre-commit

```diff
@@ -0,0 +1,2 @@
+cd frontend
+npx lint-staged
```

Comment:
Open to this but I think it's worth discussing with the team whether we want linting running on pre-commit hooks.

Reply:
Actually, since it's frontend only, this can be a decision for just me, you and Mike. What are your thoughts @mic-smith?

Reply:
Maybe having the linting as a pre-push hook might be better?

Reply:
Can do, I have no strong feelings towards either